app/assets/stylesheets/components/_label.sass

```diff
@@ -0,0 +1,6 @@
+$Label-color-text: Theme-color(text, label)
+$Label-fontSize: Theme-fontSize(text, small)
+
+.Label
+  color: $Label-color-text
+  font-size: $Label-fontSize
```

Comment:
If you're enforcing the font-size perhaps it's a good idea to enforce the line-height as well